routes/index.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   'get /agolworkers': 'getQueueCounts',
   'get /agol/:id': 'find',
   'get /agol/:id/*': 'setHostKey',
+  'put /agol/:id/*': 'setHostKey',
```

Comment:
I get it, just put is rarely used since it wasn't supported by browsers for a long time. I think modern browsers now support put and delete methods, but I'd say post fallback is a good idea. What do you think?